Source/help.cpp

```diff
@@ -13,201 +15,157 @@
-	   "|"
-	   "$Gold|"
-	   "You can select a specific amount of gold to drop by"
-	   "right-clicking on a pile of gold in your inventory.|"
-	   "|"
-	   "$Skills & Spells:|"
-	   "You can access your list of skills and spells by left-clicking on "
+	   "the corresponding number or right-clicking on the item."),
+	"",
+	N_("$Gold"),
+	N_("You can select a specific amount of gold to drop by"
+	   "right-clicking on a pile of gold in your inventory."),
+	"",
+	N_("$Skills & Spells:"),
+	N_("You can access your list of skills and spells by left-clicking on "
 	   "the 'SPELLS' button in the interface bar. Memorized spells and "
 	   "those available through staffs are listed here. Left-clicking on "
 	   "the spell you wish to cast will ready the spell. A readied spell "
-	   "may be cast by simply right-clicking in the play area.|"
-	   "|"
-	   "$Using the Speedbook for Spells|"
-	   "Left-clicking on the 'readied spell' button will open the 'Speedbook' "
+	   "may be cast by simply right-clicking in the play area."),
+	"",
+	N_("$Using the Speedbook for Spells"),
+	N_("Left-clicking on the 'readied spell' button will open the 'Speedbook' "
 	   "which allows you to select a skill or spell for immediate use.  "
 	   "To use a readied skill or spell, simply right-click in the main play "
-	   "area.|"
-	   "Shift + Left-clicking on the 'select current spell' button will clear the readied spell|"
-	   "|"
-	   "$Setting Spell Hotkeys|"
-	   "You can assign up to four Hotkeys for skills, spells or scrolls.  "
+	   "area."),
+	N_("Shift + Left-clicking on the 'select current spell' button will clear the readied spell"),
+	"",
+	N_("$Setting Spell Hotkeys"),
+	N_("You can assign up to four Hotkeys for skills, spells or scrolls.  "
 	   "Start by opening the 'speedbook' as described in the section above. "
 	   "Press the F5, F6, F7 or F8 keys after highlighting the spell you "
-	   "wish to assign.|"
-	   "|"
-	   "$Spell Books|"
-	   "Reading more than one book increases your knowledge of that "
-	   "spell, allowing you to cast the spell more effectively.|"
-	   "&")
+	   "wish to assign."),
+	"",
+	N_("$Spell Books"),
+	N_("Reading more than one book increases your knowledge of that "
+	   "spell, allowing you to cast the spell more effectively."),
 };
 
+std::vector<std::string> HelpTextLines;
+
 void InitHelp()
 {
 	helpflag = false;
-}
+	char tempstr[512];
 
-static void DrawHelpLine(const CelOutputBuffer &out, int x, int y, char *text, uint16_t style)
-{
-	const int sx = x + 32 + PANEL_X;
-	const int sy = y * 12 + 44 + UI_OFFSET_Y;
-	DrawString(out, text, { sx, sy, 577, 0 }, style);
+	for (auto text : gszHelpText) {
+		strcpy(tempstr, _(text));
+
+		WordWrapGameString(tempstr, 577);
+		const std::string_view paragraph = tempstr;
+
+		size_t previous = 0;
+		while (true) {
+			size_t next = paragraph.find("\n", previous);
+			HelpTextLines.push_back(paragraph.substr(previous, next));

```

Comment:
This can be `emplace_back` to call the `std::string(std::string_view)` constructor

Reply:
Ok :)